cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/SingleBrokerStats.java

```diff
@@ -45,6 +54,15 @@ BasicStats basicStats() {
     return _basicStats;
   }
 
+  Map<String, Double> utilByDisk() {
+    return _utilByDisk;
+  }
+
+  double pctForDisk(String logdir) {
+    return _capacityByDisk.get(logdir) < 0 ? -1 :
+                                             _utilByDisk.get(logdir) * 100 / _capacityByDisk.get(logdir);
```

Comment:
What if the given `logdir` does not exist in `_utilByDisk` or `_capacityByDisk` or these fields are `null`?